python/foglamp/services/core/api/scheduler.py

```diff
@@ -244,12 +254,13 @@ def _extract_args(data, curr_value):
         schedules.append({
             'id': str(sch.schedule_id),
             'name': sch.name,
-            'processName': sch.process_name,
+            'process_name': sch.process_name,
```

Comment:
not sure, this affects REST API response? snake_case vs camelCase (response now use **camelCase**)